index.js

```diff
@@ -50,7 +50,7 @@ module.exports = function (opts) {
 		return Promise.resolve();
 	}
 
-	var child = childProcess.spawn('npm', ['install', '--save-dev', 'ava'], {
+	var child = spawn('npm', ['install', '--save-dev', 'ava'], {
```

Comment:
Can you add a `TODO` comment here about using https://github.com/sindresorhus/execa when we eventually target Node.js >=4